File: ArchitectureComponentsMVVM/app/src/main/java/com/andr/mvvm/MainActivity.java
package com.andr.mvvm;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import com.andr.mvvm.views.UserInfoFragment;
import com.andr.mvvm.views.UserListFragment;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        getSupportFragmentManager().beginTransaction().replace(R.id.container_first,new UserListFragment()).commit();
        getSupportFragmentManager().beginTransaction().replace(R.id.container_second,new UserInfoFragment()).commit();
    }
}

File: MVVM_Kotlin/app/src/main/java/com/mvvm/kot/MainActivity.kt
package com.mvvm.kot

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import com.mvvm.kot.Views.UserInfoFragment
import com.mvvm.kot.Views.UserListFragment

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        supportFragmentManager.beginTransaction().replace(R.id.container_first, UserListFragment()).commit()
        supportFragmentManager.beginTransaction().replace(R.id.container_second, UserInfoFragment()).commit()
    }
}
